plugins/woocommerce/client/admin/client/marketplace/README.md

```diff
@@ -14,6 +14,32 @@ The page contains two parts, the WooCommerce.com marketplace and a list of produ
 
 ### Marketplace API
 
+The Marketplace API handles interactions with WooCommerce.com and local product data. Here are the main endpoints:
+
+1. **Product Discovery**:
+   - `/wc/v3/marketplace/featured`: Gets curated extensions and themes for the Discover page.
```

Comment:
Is this a real endpoint? If I'm remembering right, the endpoint for the featured products is `wp-json/wccom-extensions/2.0/featured`.

Reply:
If you remember, `/wc/v3/marketplace/featured` is actually a proxy API built directly in core that makes calls to the `wp-json/wccom-extensions/2.0/featured` on WCCOM that you mentioned. This specific quirk is the exact reason I made these docs changes. It's confusing every time I work with these APIs.